src/firefly/js/visualize/ChartsTableViewPanel.jsx

```diff
@@ -47,19 +43,34 @@ import FILTER_IN from 'html/images/icons-2014/24x24_FilterAdd.png';
 import CLEAR_FILTERS from 'html/images/icons-2014/24x24_FilterOff_Circle.png';
 import LOADING from 'html/images/gxt/loading.gif';
 
+const selectionBtnStyle = {padding: '0 5px', cursor: 'pointer'};
 
-const OPTIONS_WIDTH = 330;
-
-const selectionBtnStyle = {verticalAlign: 'top', paddingLeft: 20, cursor: 'pointer'};
-
+/*
+ CSS for left positioned options
+ left: 1
+ borderRight: '1px solid #CCC',
+ boxShadow: '2px 0 5px rgba(0,0,0,0.50)'
+ */
+const chartOptionsStyle = {
+    position: 'absolute',
+    padding: '0 5px',
+    top: 1,
+    right: 1,
+    bottom: 2,
+    backgroundColor: 'beige',
+    borderLeft: '1px solid #CCC',
+    zIndex: 100,
+    flex: '0 0 auto',
+    overflow:'auto',
+    boxShadow: '-2px 0 2px 0 rgba(0,0,0,0.50)'
```

Comment:
i see lots of inline styling in this file.  having them in a css file may look cleaner.